src/DynamoCore/Graph/Nodes/PortModel.cs

```diff
@@ -408,7 +408,7 @@ internal string GetInputPortType()
                 string type;
                 // In the case of a node for an instance method, the first port
                 // type is the declaring class type of the method itself.
-                if (fd.Type == FunctionType.InstanceMethod)
+                if (fd.Type == FunctionType.InstanceMethod || fd.Type == FunctionType.InstanceProperty)
```

Comment:
@aparajit-pratap I guess I am not too familiar with `FunctionType` enum, do we need to consider other options here?

Reply:
The only other option I need to check is `Constructor`. Let me check.

Reply:
I have verified that this works well with constructors too.